### ISSUE TITLE
Easier way to set numerical values on an unmodified Taranis

### ISSUE THREAD

**Issue by pindeslandes** [opentx/opentx#1606]
This proposal could be related with #1351 but I am not sure because I wish to avoid any hard modification (no encoder in the place of a pot).  The advantages being that it would satisfy a large majority of average users (well, I hope).
That’s why I open a new issue.

So, in my dreams, it would be possible to set values using S1 or S2 pot.

I imagine it could be working this way:

0) Of course, before you start, S1 and S2 can be used as usual. No change here.
1) You have a value to set, so you highlight it in the relevant menu and press ENTER (as usual).
2) As soon as you move S1 or S2 (see note 2), the value corresponding to its position shows up and replaces the initial value (that would be the new feature, let’s call it ‘pot mode’).
3) Press ENTER to record or, if you need finer tuning (because, for example, it might be difficult to reach the exact decimal of a subtrim with a pot), use +/- keys and then ENTER as usual (see note 4).
4) Then, of course again, S1 and S2 are available back to their regular role (see note 5).

Note 1: The philosophy is ‘begin with fast and rough setting with a pot and end with fine tuning with keys if you need to’ but it would always be possible to skip the pot mode directly to the key mode. Just don’t touch the pots after step 1 and it’s the same as before, you won’t see any difference.
Note 2: You don’t have to choose between S1 and S2 somewhere in the radio general settings: the first you touch is the winner. So if you touch a second pot while you are setting a value with the first you moved, the second pot shouldn’t have any effect on it.
Note 3: The throw of the pot should automatically match the range of the parameter you are setting (+/- 1024 for a GVAR, +/- 150 for a servo throw (if extended) etc.). So may be, at least for setting of such parameters as GVARs, it would help if the pot had an exponential effect (less sensitive around 0), only the time of the setting, of course.
Note 4: It might be good to deactivate the pot mode as soon as a +/- key is used to avoid unexpected effects (for example, unwillingly touching a pot and unsetting a subtrim you are just being tuning precisely with +/- keys). This would be consistent with the philosophy of note 1.
Note 5: If the pot used for the setting is already used in the model, it may lead to difficulties (I am not sure of that because, honestly, I can’t find a case where it would create a real problem; the worst I can imagine is creating loops but it can already happen today in many cases, and as long as it doesn’t freeze the Tx, it’s OK I think). So may be, as a precaution, the pot should be deactivated for any other purpose the time of the setting.
Note 6: It may be imaginable to use not only S1 or S2 but also any of the physical inputs. But I don’t think it would be better because it doesn’t make things a lot easier and it certainly means more code to write, more new bugs to come with it, more memory needed, and probably messier operation.

Long to explain but easy to operate.
